specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-11-01/examples/GetExtension.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "subId1",
+    "resourceGroupName": "rg1",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterResourceName": "connectedClusters",
+    "extensionName": "ClusterMonitor",
+    "api-version": "2022-11-01",
+    "clusterName": "clusterName1"
+  },
+  "responses": {
+    "200": {
```

Comment:
Can you please add two examples?
GetExtension.json for regular extensions
GetExtensionWithPlan.json for Marketplace extensions